### ISSUE TITLE
tedge_mapper az/collectd services start running when the thin-edge device restarts

### ISSUE THREAD

**Issue by PradeepKiruvale** [thin-edge/thin-edge.io#1631]
**Describe the bug**
When the `thin-edge` device is connected to `c8y` and restarts
The `tedge_mapper az and collectd` start running. These are not supposed to run by default.

**To Reproduce**
1. `sudo tedge connect c8y` (just restarting the system)
2. Reboot the system
3. ps -ef | grep tedge 
    You can see tedge_mapper c8y, az, and collectd running.

**Expected behavior**
The `tedge_mapper az/collectd` are not supposed to start, because they are not started earlier and not connected to azure cloud.

**Screenshots**

**Environment (please complete the following information):**
 - OS: Ubunutu
 - Hardware x64
 - System-Architecture [e.g. result of "uname -a"]
 - thin-edge.io version [e.g. 0.8.1]

**Additional context**

**Comment by reubenmiller** [comment]
@PradeepKiruvale Can you provide the details on how you installed tedge (e.g. manually or via the script)?

Or did you already find the code which was enabling the systemd services?

**Comment by PradeepKiruvale** [comment]
> @PradeepKiruvale Can you provide the details on how you installed tedge (e.g. manually or via the script)?
I did install using the `dpkg` locally built `Debian` packages. I feel the something will be done by the script as well.
> 
> Or did you already find the code which was enabling the systemd services?

Not yet, I will have to find out.

**Comment by rina23q** [comment]
@PradeepKiruvale and I found that this bug start occurring since **0.8.0** release. 0.7.7 release doesn't have this behaviour. I am doubting because of this change. https://github.com/thin-edge/thin-edge.io/pull/1509/files#diff-408c572b7e6364e06e37f3d7dd2f9aad57f4706c5b8d4026d76c35b7b3271c2b

Probably these 3 lines should not have been removed from Cargo.toml of tedge-mapper.
```toml
enable = false
start = false
stop-on-upgrade = false
```
